newIDE/app/src/Export/BrowserExporters/BrowserOnlineWebExport.js

```diff
@@ -61,7 +64,11 @@ export const browserOnlineWebExportPipeline: ExportPipeline<
 
   renderHeader: () => <ExplanationHeader />,
 
-  renderLaunchButtonLabel: () => <Trans>Publish online</Trans>,
+  renderLaunchButtonLabel: () => <Trans>Generate link</Trans>,
+
+  customStepsProgress: (build: ?Build, loading: boolean) => (
```

Comment:
Call this `renderCustomStepsProgress` (a function should be a verb, unless it's a Component, and this verb is often `render` for things returning `React.Node`s)